endpointmanager/pkg/chplquerier/vendorquerier_test.go

```diff
@@ -339,32 +351,52 @@ func Test_getVendorJSON(t *testing.T) {
 	th.Assert(t, actualVendorsReceived == expectedVendorsReceived, fmt.Sprintf("Expected to receive %d vendors Actually received %d vendors.", expectedVendorsReceived, actualVendorsReceived))
 
 	// test context ended.
-	// also checks what happens when an http request fails
 
 	tc, err = basicVendorTestClient()
 	th.Assert(t, err == nil, err)
 	defer tc.Close()
 
+	hook := logtest.NewGlobal()
+	expectedErr := "Got error:\nmaking the GET request to the CHPL server failed:"
+
 	ctx, cancel := context.WithCancel(context.Background())
 	cancel()
 
-	_, err = getVendorJSON(ctx, &(tc.Client))
-	switch reqErr := errors.Cause(err).(type) {
-	case *url.Error:
-		th.Assert(t, reqErr.Err == context.Canceled, "Expected error stating that context was canceled")
-	default:
-		t.Fatal("Expected context canceled error")
+	_, err = getVendorJSON(ctx, &(tc.Client), "")
+	th.Assert(t, err == nil, err)
+
+	// expect presence of a log message
+	found := false
+	for i := range hook.Entries {
+		if strings.Contains(hook.Entries[i].Message, expectedErr) {
+			found = true
+			break
+		}
 	}
+	th.Assert(t, found, "expected a context canceled error to be logged")
 
 	// test http status != 200
 
 	tc = th.NewTestClientWith404()
 	defer tc.Close()
 
+	hook = logtest.NewGlobal()
+	expectedErr = "CHPL request responded with status: 404 Not Found"
+
```

Comment:
Same comment as before, maybe just to keep things consistent, move the lines with variables hook and expectedErr to above the lines where the test client is made, since this is the ordering in the other files containing these kinds of tests.